README.md

```
@@ -1,22 +1,29 @@
+  3. 右侧键盘屏幕上的GIF动画被移除，这将显著降低右侧键盘的功耗。
+
+> 如果您的键盘于2025年8月22之前更新，请更新最新的固件。
+>
 
-> 如果您的键盘于10月24日之前更新，请更新最新的固件。
-> 
----
-# 联系我
+## 联系我
 
 如需3D打印的模型文件或者键盘有任何异常和故障，请联系380465425@qq.com
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)